### PR TITLE
17107 Happy path Cypress e2e test for COVID-19 vaccine prep tool

### PR DESCRIPTION
**Pull request by ddzz**
## Description
Cypress e2e test for signing up for the COVID-19 vaccine prep form without being logged in. [ZenHub ticket](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/17107).

## Testing done
- All Cypress tests pass on CI
- New Cypress test passes locally

## Acceptance criteria
- [ ] Cypress tests pass locally and on CI

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by billfienberg — COMMENTED** [review]
image

It looks like your test is failing.

**Comment by ddzz** [comment]
> image
> 
> It looks like your test is failing.

Thanks for reviewing, I just pushed up a fix that should pass.

**Review by billfienberg — APPROVED** [review]

**Review by pjhill — APPROVED** [review]
LGTM!

File: src/applications/coronavirus-vaccination/tests/e2e/signup.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      cy.get('#root_vaccineInterest-label').contains(
+        'Do you plan to get a COVID-19 vaccine when one is available to you?',
+      );
+      cy.get('#root_vaccineInterest_0').check();
+
+      cy.route('POST', '**/covid_vaccine/v0/registration', {
+        status: 200,
+      }).as('response');
+
+      cy.get('.usa-button').contains('Submit form');
```
Comment (1Copenut) [inline src/applications/coronavirus-vaccination/tests/e2e/signup.cypress.spec.js:78]: I'd recommend one more `cy.axeCheck();` here with the form filled out. It does feel right at the edge of superfluous, but gives us a fuller picture of accessibility with a pristine form, and one that has data entered.

**Review by 1Copenut — DISMISSED** [review]

File: src/applications/coronavirus-vaccination/tests/e2e/signup.cypress.spec.js (hunk 1 above)
Comment (ddzz) [reply src/applications/coronavirus-vaccination/tests/e2e/signup.cypress.spec.js:79]: @1Copenut thanks for reviewing. Just added the second `axecheck()`.

**Review by saneshark — APPROVED** [review]
LGTM

**Review by 1Copenut — APPROVED** [review]
👍 LGTM!

**Comment by pjhill** [comment]
Completes the ticket -- [Automate High Value Test Cases](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/17107)

**Pull request merged**